pytest.ini

```diff
@@ -0,0 +1,4 @@
+# pytest.ini
+[pytest]
+python_files= test_*.py
```

Comment:
This _should_ be unnecessary, because Pytest looks for `test_*.py` or `*_test.py` by default - was this maybe to try and fix it failing to find tests that you then fixed by adding the `__init__.py`?
No harm in leaving it, but worth wondering briefly whether it's useful!